Documentation/deploy-master.md

```diff
@@ -33,11 +33,12 @@ $ sudo chmod 600 /etc/kubernetes/ssl/*-key.pem
 $ sudo chown root:root /etc/kubernetes/ssl/*-key.pem
 ```
 
-### flannel Configuration
+### Network Configuration
+Networking is provided by Flannel and Calico.
+* [flannel][flannel-docs] provides a software-defined overlay network for routing traffic to/from the [pods][pod-overview]
+* [Calico][calico-docs] secures the overlay network by restricting traffic to/from the [Pod]s based on fine-grained network policy.
 
-[flannel][flannel-docs] provides a key Kubernetes networking capability &mdash; a software-defined overlay network to manage routing of the [Pod][pod-overview] network.
-
-*Note:* If the pod-network is being managed independently of flannel, this step can be skipped. See [kubernetes networking](kubernetes-networking.md) for more detail.
+*Note:* If the pod-network is being managed independently of flannel, then the flannel parts of this guide can be skipped. It's recommended that Calico is still used for providing network policy. See [kubernetes networking](kubernetes-networking.md) for more detail.
```

Comment:
replace second sentence: "In this case, Calico may still be used for network policy."
Capitalize "Kubernetes"